R/style-roxygen-examples.R

```diff
@@ -59,10 +51,10 @@ style_roxygen_code_examples_one <- function(path) {
 }
 
 
-remove_roxygen_mask <- function() {
-  NULL
+remove_roxygen_mask <- function(text) {
+  sub(pattern = "#'(\\s)?", "", text)
```

Comment:
Not sure if we should use `^#'\\s*` here too, since we don't want to replace the first `#'` in the string, but only if the string starts with that pattern.